### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by kumarp**
Added documentation. Needs to be filled out with: References for technologies/concepts, images describing structures.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,31 @@ Demonstration of how behavior driven development can be applied in a containeriz
...
+
+By combining the power of containerized environments and core DevOps concepts, we are able to control code quality while maintaining a rapid turnaround time to deploying the latest code additions to higher environments.
+
+## Core Technologies Demonstrated
+
+OpenShift Enterprise
+Docker
+Jenkins
+Cucumber
+Red Hat JBoss BPMS
```
Comment (sabre1041) [inline README.md]: We are running BRMS, not bpms in this demo

File: README.md [hunk 2]
```diff
@@ -67,4 +92,4 @@ spec:
     server: localhost
   persistentVolumeReclaimPolicy: Recycle
 PV
```
Comment (sabre1041) [inline README.md:94]: We'll have to expand upon the system requirements moving forward as 5 PV's will be required

File: documentation/DemonstrationScript.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+Demo Agenda
+===========
+This section provides a high-level agenda for the demo. The paranthesis after each discuss what concepts are highlighted in that portion.
+
+1. Discuss what the application is. (BPMS, Behavior Driven Development)
+2. Show that the application is up and running in a stable state. (OpenShift Enterprise, Jenkins, CI/CD, DevOps)
+3. Change the business requirments. (Behavior Driven Development)
+4. Change the logic. (BPMS)
```
Comment (sabre1041) [inline documentation/DemonstrationScript.txt]: Same as comment in README. bpms->brms

**Comment by kumarp** [comment]
Made updates of BPMS -> BRMS

**Comment by sabre1041** [comment]
Merging first round

**Pull request merged**